2018-12-11 11:29:52 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109210> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:29:53 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109211> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:29:54 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109212> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:29:55 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109213> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:29:57 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109214> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:29:58 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109215> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:29:59 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109216> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:01 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109217> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:02 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109218> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:03 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109219> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:04 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109220> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:06 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109221> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:07 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109222> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:08 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109223> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:09 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109224> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:11 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109225> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:12 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109226> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:14 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109227> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:30:15 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109228> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 30, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:31:19 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109210> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 31, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:31:20 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109211> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 31, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:31:22 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=109212> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 31, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:31:52 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=114692> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 31, in parse
    data = dict_data['result']['data']
KeyError: 'result'
2018-12-11 11:31:53 [scrapy.core.scraper] ERROR: Spider error processing <GET http://i.sporttery.cn/api/fb_match_info/get_asia/?f_callback=asia_tb&mid=114693> (referer: None)
Traceback (most recent call last):
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/.virtualenvs/pachong/lib/python3.5/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/Users/admin/Desktop/spider/t_spider/dl_asia/real_time/spiders/ya.py", line 31, in parse
    data = dict_data['result']['data']
KeyError: 'result'
